ros2controlcli/ros2controlcli/verb/load_configure_controller.py

```diff
@@ -31,8 +30,8 @@ def add_arguments(self, parser, cli_name):
         add_controller_mgr_parsers(parser)
 
     def main(self, *, args):
+        print("deprecated warning: Please use either 'load --set_state' or 'set_state'")
```

Comment:
probably not the best line to raise this but somewhat relevant and at least it shows the command...

what do you think if we called the `command` for setting the state `set_state` but used `state` as the argument when combined with other commands? aka `ros2 control load joint_state_broadcaster --state start`

Reply:
> probably not the best line to raise this but somewhat relevant and at least it shows the command...

fair, but what's a better suited place for it?


I'll change it to `--state`.